### PR TITLE
[platform/device] - Update sonic-platform-module-cel and device configuration for haliburton 

### PR DESCRIPTION
**Pull request by mudsut4ke**
**- What I did**
- Fixed Haliburton fancontrol issue
- Add sfputil for Haliburton device 
- Add psuutil for Haliburton device
- Add sensor driver and sensor.conf for Haliburton device 
- Merged missing lastest dx010 code version
- Fixed dx010 issue (#1896) 

Signed-off-by: Wirut Getbamrung wgetbumr@celestica.com

### CODE REVIEW

File: platform/broadcom/sonic-platform-modules-cel/debian/platform-modules-dx010.init [hunk 1]
```diff
@@ -75,11 +56,11 @@ start)
         echo 24lc64t 0x50 > /sys/bus/i2c/devices/i2c-12/new_device
 
         # Attach temperature sensors
-        echo lm75b 0x48 > /sys/bus/i2c/devices/i2c-5/new_device
-        echo lm75b 0x49 > /sys/bus/i2c/devices/i2c-6/new_device
-        echo lm75b 0x4a > /sys/bus/i2c/devices/i2c-7/new_device
-        echo lm75b 0x48 > /sys/bus/i2c/devices/i2c-14/new_device
-        echo lm75b 0x4e > /sys/bus/i2c/devices/i2c-15/new_device
+        echo dx010_lm75b 0x48 > /sys/bus/i2c/devices/i2c-5/new_device
```
Comment (lguohan) [inline platform/broadcom/sonic-platform-modules-cel/debian/platform-modules-dx010.init]: why can you use original lm75b?

File: platform/broadcom/sonic-platform-modules-cel/dx010/modules/dx010-fan-led.c [hunk 2]
```diff
@@ -0,0 +1,209 @@
+/*
+ * dx010-fan-led.c - Kernel module for Seastone DX010 fan LEDs
```
Comment (lguohan) [inline platform/broadcom/sonic-platform-modules-cel/dx010/modules/dx010-fan-led.c]: the fan led control should be left to user space, can you revert this dx010-fan-led change?

File: platform/broadcom/sonic-platform-modules-cel/dx010/modules/lm75.c [hunk 3]
```diff
@@ -0,0 +1,553 @@
+/*
+ * lm75.c - Part of lm_sensors, Linux kernel modules for hardware
```
Comment (lguohan) [inline platform/broadcom/sonic-platform-modules-cel/dx010/modules/lm75.c]: why not use original lm75 in the kernel, why build your own?

**Comment by lguohan** [comment]
please remove binary file. lpc_cpld_x64_64

File: platform/broadcom/sonic-platform-modules-cel/debian/platform-modules-haliburton.init [hunk 4]
```diff
@@ -16,9 +16,10 @@ start)
         echo -n "Setting up board... "
 
         depmod -a
-        modprobe i2c-dev
-        modprobe i2c-mux-pca954x
+	modprobe smc
 
+        (cd /usr/local/etc && ./lpc_cpld_x64_64 blu w 0x222 0x1f)
```
Comment (lguohan) [inline platform/broadcom/sonic-platform-modules-cel/debian/platform-modules-haliburton.init]: cannot check-in a binary file.

File: platform/broadcom/sonic-platform-modules-cel/dx010/modules/dx010-fan-led.c (hunk 2 above)
Comment (mudsut4ke) [reply platform/broadcom/sonic-platform-modules-cel/dx010/modules/dx010-fan-led.c]: @pphuchar Please help to check this one.

File: platform/broadcom/sonic-platform-modules-cel/debian/platform-modules-haliburton.init (hunk 4 above)
Comment (mudsut4ke) [reply platform/broadcom/sonic-platform-modules-cel/debian/platform-modules-haliburton.init]: Our team need to use this tool to reset phy port. 
How can I use this tool ?

File: platform/broadcom/sonic-platform-modules-cel/debian/platform-modules-dx010.init (hunk 1 above)
Comment (mudsut4ke) [reply platform/broadcom/sonic-platform-modules-cel/debian/platform-modules-dx010.init]: I will review it again.

**Comment by lguohan** [comment]
retest this please

File: platform/broadcom/sonic-platform-modules-cel/debian/platform-modules-haliburton.init [hunk 5]
```diff
@@ -16,9 +16,10 @@ start)
         echo -n "Setting up board... "
 
         depmod -a
```
Comment (lguohan) [inline platform/broadcom/sonic-platform-modules-cel/debian/platform-modules-haliburton.init]: we should not have depmod -a in the init script, depmod -a should be in the postinst script.

File: platform/broadcom/sonic-platform-modules-cel/dx010/modules/dx010-fan-led.c (hunk 2 above)
Comment (lguohan) [reply platform/broadcom/sonic-platform-modules-cel/dx010/modules/dx010-fan-led.c]: current ask is just revert this to previous version, we can deal with the user-space fan led control later. Not in this PR.

**Comment by mudsut4ke** [comment]
Move to #1975

**Pull request closed without merging**